### ISSUE TITLE
[BUG] - towers attack castle far from castle

### ISSUE THREAD

**Issue by AeroFoxxo** [Nexyl-Roblox/Issues#25]
### Description

the towers will attack far -ish from castle 

### Reproduction steps

```bash
1. start round
2. wait for enemies to get to castle (more clear when many animies)
3. watch some attack the castle infront of the castle
```

### Screenshots

```bash
![DESCRIPTION](LINK.png)
https://imgur.com/a/E0Pr5bl far from home (attacks castle before at castle (my towers dont work its not the towers fault))
```

**Comment by AeroFoxxo** [comment]
medium severity maybe

**Comment by AeroFoxxo** [comment]
ok some of them stop walking for a second so theres also that problem but it might just be roblox

**Comment by gavinostler** [comment]
Towers have been fixed as of #22
